pkgs/tools/misc/tvnamer/default.nix

```diff
@@ -4,16 +4,20 @@
 
 let
   python' = python3.override {
-    packageOverrides = self: super: rec {
+    packageOverrides = final: prev: rec {
       # tvdb_api v3.1.0 has a hard requirement on requests-cache < 0.6
-      requests-cache = super.requests-cache.overridePythonAttrs (super: rec {
+      requests-cache = prev.requests-cache.overridePythonAttrs (o: rec {
```

Comment:
```suggestion
      requests-cache = prev.requests-cache.overridePythonAttrs (oldAttrs: rec {
```

Reply:
Applied and had to change one more instance. This is way more clear, thanks!